src/psyclone/domain/lfric/lfric_arg_descriptor.py

```diff
@@ -738,7 +738,7 @@ def function_spaces(self):
     @property
     def vector_size(self):
         '''
-        Returns the vector size of the argument. This will be 1 if *n
+        Returns the vector size of the argument. This will be 1 if '\*n'
```

Comment:
pycodestyle complains here ...

lfric_arg_descriptor.py:741:69: W605 invalid escape sequence '\*'

Presumably pycodestyle does this in error seeing as we're in a string?

Reply:
Yes, I thought it should be an error but apparently it isn't. I've found a better solution now.